### PR TITLE
Fix `Any` evaluating even if entry is nil

### PR DESCRIPTION
**Pull request by Kangaroux**
- Added nil entry check to `Any`
- Refactored some nil checks that weren't consistent
- Added newlines after nil conditional (some funcs already had a newline but not all)

### CODE REVIEW

File: logger.go [hunk 1]
```diff
@@ -690,6 +690,7 @@ func (e *Entry) Time(key string, t time.Time) *Entry {
 	if e == nil {
 		return nil
 	}
+
```
Comment (ccoVeille) [inline logger.go:693]: While this change (and the other ones new line changes) is good for readability, I would have put them in a dedicated commit (or PR)

It would be easier to review

**Review by ccoVeille — APPROVED** [review]

**Comment by phuslu** [comment]
thanks!

**Comment by phuslu** [comment]
v1.0.107 tagged, 😄
